libs/common/src/typegraph/mod.rs

```diff
@@ -33,6 +33,7 @@ pub struct Typegraph {
     pub runtimes: Vec<TGRuntime>,
     pub policies: Vec<Policy>,
     pub meta: TypeMeta,
+    pub random_seed: Option<u32>,
```

Comment:
I suggest to move this in the `meta` field (`TypeMeta` struct).

Reply:
ok, I guess that makes more sense. will do that.

Reply:
Done.